README.md

```diff
@@ -681,6 +681,15 @@ And finally, as part of the site build process, we processed all pending notific
         "processed": false
     }
     ```
+
+### Error logging
+
+For the error logging for the whole site and background processes this project uses [Rollbar](https://rollbar.com/seanbmcgregor).
```

Comment:
I would remove specific mentions of Sean's rollbar project and make the description more generic 🤔

Reply:
done!